src/v2i-hub/CswPlugin/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
 	"name":"CSW",
 	"description":"Provides Curve Speed Warning (CSW).",
-	"version":"@PROJECT_VERSION@",
+	"version":"5.0",
```

Comment:
Can we leave the version value as it was?  We would like to avoid hardcode "5.0" here. I just did a global search, it looks like all the manifest.json is using this "PROJECT_VERSION" variable.

Reply:
Is there a reason you update this to "5.0"?

Reply:
Lets keep this as a running variable.

Reply:
This was an oversight. For some reason when I built it, that got changed in all the manifest jsons and I changed it back for most. Must have missed this one. Will fix it.